src/components/ClusterNodes/clusterNodes.scss

```diff
@@ -0,0 +1,221 @@
+.cluster-list {
+    &.no-result-container {
+        height: calc(100vh - 48px);
+        overflow: hidden;
+    }
+    .cluster-list-row {
+        display: grid;
+        grid-template-columns: auto 150px 50px 90px 100px 100px 120px;
+        column-gap: 16px;
+        .cluster-status {
+            height: 12px;
+            width: 12px;
+        }
+    }
+}
+.node-list {
+    .width-50 {
+        width: 50%;
+    }
+    .width-25 {
+        width: 25%;
```

Comment:
Suggestion - can we move these to base.scss so that it can be used by other comps as well?

Reply:
done